components/frontend/src/utils.jsx

```diff
@@ -421,7 +421,7 @@ export function dropdownOptions(options) {
 
 export function referenceDocumentationURL(name) {
     // Return a URL to the documentation for the metric/subject/source name
-    const slug = `${name?.toLowerCase().replaceAll(" ", "-").replaceAll(/[()/]/g, "")}`
+    const slug = `${name?.toLowerCase().replaceAll(" ", "-").replaceAll("_", "-").replaceAll(/[()/]/g, "")}`
```

Comment:
Is this `replaceAll` a regex? Perhaps the first can then be replaced by something like `replaceAll([\s_]), "-")`?

Reply:
The first argument can be a regex yes. In JavaScript strings between slashes are regular expression literals.

Reply:
Merged `replaceAll(" ", "-").replaceAll("_", "-")` into `replaceAll(/[\s_]/g, "-")`.